scripts/oneoff/export-dimension-titles-as-csv.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+
+import sys
+import csv
+
+from application import db
+from application.factory import create_app
+from application.config import DevConfig
+
+sys.path.insert(0, ".")
```

Comment:
Hmm, what do we need this for?

Reply:
No idea... seems to be required before the `import` statements. Have moved it.